modules/telco-upgrade-starting-the-y-stream-control-plane-upgrade.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-starting-the-y-stream-control-plane-upgrade_{context}"]
+= Starting the y-stream control plane upgrade
+
+Once you have determined the full new release that you are moving to, you can run the `oc adm upgrade –to=x.y.z` command.
+
+.Procedure
+* Start the y-stream control plane upgrade.
+For example, run the following command:
++
+[source,console]
+----
+$ oc adm upgrade --to=4.16.14
+----
++
+.Example output
+[source,console]
+----
+Requested update to 4.16.14
+----
++
+You might move to a z-stream release that has potential issues with platforms other than the one you are running on.
+The following example shows a potential problem for cluster upgrades on Microsoft Azure:
++
+[source,console]
+----
+$ oc adm upgrade --to=4.16.15
+----
++
+.Example output
+[source,console]
+----
+error: the update 4.16.15 is not one of the recommended updates, but is available as a conditional update. To accept the Recommended=Unknown risk and to proceed with update use --allow-not-recommended.
+  Reason: EvaluationFailed
+  Message: Exposure to AzureRegistryImagePreservation is unknown due to an evaluation failure: invalid PromQL result length must be one, but is 0
+  In Azure clusters, the in-cluster image registry may fail to preserve images on update. https://issues.redhat.com/browse/IR-461
+----
++
+[source,console]
+----
+$ oc adm upgrade --to=4.16.15 --allow-not-recommended
```

Comment:
This looks like it's telling them to allow them to move to a not recommended version. This should likely be a separate step at least giving them some info before running it. Perhaps some of the warning like what is shown in the output below.

Reply:
Added a note clarifying that there is no issue for bare-metal clusters